test/ui-test/testSuites/suite_settings/tst_userIdentity/test.feature

```diff
@@ -9,6 +9,7 @@ Feature: User Identity
         And the user signs up with username "tester123" and password "TesTEr16843/!@00"
         And the user lands on the signed in app
 
+    @mayfail
```

Comment:
Please, remember to add a task for refactoring this test!!!

Reply:
Created/added under https://github.com/status-im/status-desktop/issues/9909

Reply:
Thanks!!